### PR TITLE
Loop Folder Directory for Images

### LINKED ISSUE #77: Label all images in a directory
Currently, labelme can be called using -O to preset the output file and the actual image to label as an argument. Using these options one could write a script that iterates over all images in a directory to label them all.

To make labeling all images in a directory more convenient, I think it would be helpful to be able to 'open dir' from the file menu and show a list of the images in the selected directory. Ideally the list would also provide some feedback whether the image is labeled yet.

### PR DESCRIPTION
**Pull request by seahawks8**
Close #77 

Resources.qrc
 - added the next image and previous image icons

Icons folder
 - added two png images for the file menu

 App.py
screen shot 2018-03-24 at 11 58 58 pm

 - added ability to open file directory
 - load images in per file directory, go to next and previous images
 - check if json exists for image file, if so load the file.
 - open save dialog when saving or overwritting the json file.

### CODE REVIEW

File: labelme/app.py [hunk 1]
```diff
@@ -109,6 +109,9 @@ def __init__(self, filename=None, output=None, store_data=True,
 
         self.labelList = QListWidget()
         self.itemsToShapes = []
+        self.imageList = []
+        self.lastOpenDir = None
+        self.dirname = None
```
Comment (wkentaro) [inline labelme/app.py]: What is `self.dirname` for?

**Comment by wkentaro** [comment]
I tested this PR and it works nicely.
Please check my comments.

**Comment by wkentaro** [comment]
## How to test

```
labelme tutorial/
```

screen shot 2018-03-27 at 0 38 26

**Comment by seahawks8** [comment]
Thanks @wkentaro, ya that variable was unused. I did not catch that. I am a bit new to GITHUB, any other steps to merge this PR?

**Comment by wkentaro** [comment]
I will fix some minor problems by my own. (for example changes maybe you reverted)

**Comment by seahawks8** [comment]
ok, sounds good. just let me know. My next branch will be making tool reading label.txt file.

**Comment by wkentaro** [comment]
LGTM 👍 @seahawks8 Thanks!

**Pull request merged**